src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -148,6 +148,7 @@
     <Compile Include="ReadOnlySpan\LastIndexOf.T.cs" />
     <Compile Include="ReadOnlySpan\LastIndexOfSequence.byte.cs" />
     <Compile Include="ReadOnlySpan\LastIndexOfSequence.T.cs" />
+    <Compile Include="ReadOnlySpan\MemoryExtensions.Overlaps.cs" />
```

Comment:
nit: I don't think we should call this Overlaps given it isn't testing the overlaps method (and we already have `ReadOnlySpan\Overlaps.cs`)

How about `ToUpperLower.cs` instead?

Reply:
will do

Reply:
done